tests/scripts/travis-before-install.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+set -ev
+
+# Install Google App Engine Python SDK
+if [[ ! -d ${GAE_PYTHONPATH} ]]; then
+  python tests/scripts/fetch_gae_sdk.py `dirname ${GAE_PYTHONPATH}`
```

Comment:
nit: wrap ${GAE_PYTHONPATH} in quotes in case it contains special characters. told to do this for all bash variables when I submitted PR into bdutil.